src/App.tsx

```diff
@@ -94,12 +145,27 @@ const App = (_props: Props) => {
         autoClose={false}
       ></ToastContainer>
       <div>
-        <div className="min-w-screen container p-4 mx-auto flex justify-between items-center">
-          <img src={INTER_LOGO} className="item" alt="Inter Logo" width="200" />
-          {address}
-          <NetPicker currentNet={walletUtils.agoricNet} />
+        <div className="min-w-screen container p-4 mx-auto flex flex-wrap justify-between items-center">
+          <img
```

Comment:
out of scope but maybe it would help for this to link to governance docs

Reply:
What's the link to the governance docs?

Reply:
https://docs.inter.trade/inter-protocol-system-documentation/protocol-and-chain-governance is the most relevant I can find, but that's not very helpful. nvm.